.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "🙏 Feature Request"
+about: Submit a feature request
+title: "[🙏]"
+labels: "🙏 feature request"
+assignees: ''
+---
```

Comment:
I don't quite understand this table, how is it meant to be used?

Reply:
Apparently, it transforms into this table with requirements to the Issue look
https://github.com/snyk/snyk/blob/master/.github/ISSUE_TEMPLATE/bug-report.md